monkey/monkey_island/cc/resources/pba_file_upload.py

```diff
@@ -43,7 +39,8 @@ def get(self, file_type):
             filename = ConfigService.get_config_value(copy.deepcopy(PBA_LINUX_FILENAME_PATH))
         else:
             filename = ConfigService.get_config_value(copy.deepcopy(PBA_WINDOWS_FILENAME_PATH))
-        return send_from_directory(FLASK_UPLOAD_PATH, filename)
+        LOG.info(f"Current dir:{os.getcwd()}")
```

Comment:
Does this log message add enough context/information to be useful? It looks like it might be better at the DEBUG level than INFO.